<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>BranchesUi</class>
 <widget class="QWidget" name="BranchesUi" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="newBranchLE" />
   </item>
   <item row="5" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2" >
    <widget class="QPushButton" name="addBranchPB" >
     <property name="toolTip" >
      <string>Add a new branch to the list</string>
     </property>
     <property name="text" >
      <string>&amp;Add</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QLabel" name="availableLB" >
     <property name="text" >
      <string>A&amp;vailable Branches:</string>
     </property>
     <property name="buddy" >
      <cstring>branchesTW</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="newBranchLA" >
     <property name="text" >
      <string>&amp;New:</string>
     </property>
     <property name="buddy" >
      <cstring>newBranchLE</cstring>
     </property>
    </widget>
   </item>
   <item rowspan="4" row="2" column="0" colspan="2" >
    <widget class="QTreeWidget" name="branchesTW" />
   </item>
   <item row="2" column="2" >
    <widget class="QPushButton" name="removePB" >
     <property name="toolTip" >
      <string>Remove the selected branch</string>
     </property>
     <property name="text" >
      <string>&amp;Remove</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" >
    <widget class="QPushButton" name="activatePB" >
     <property name="toolTip" >
      <string>Toggle the selected branch</string>
     </property>
     <property name="text" >
      <string>(&amp;De)activate</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" >
    <widget class="QPushButton" name="colorPB" >
     <property name="toolTip" >
      <string>Define or change background color</string>
     </property>
     <property name="text" >
      <string>Alter Co&amp;lor...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <includes>
  <include location="local" >qt_i18n.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
